x-pack/plugins/uptime/server/lib/requests/search/find_potential_matches.ts

```diff
@@ -43,38 +44,53 @@ const query = async (queryContext: QueryContext, searchAfter: any, size: number)
   return await queryContext.search(params);
 };
 
-const queryBody = async (queryContext: QueryContext, searchAfter: any, size: number) => {
+const queryBody = async (queryContext: QueryContext, size: number, index: number) => {
   const filters = await queryContext.dateAndCustomFilters();
 
+  const sortField = queryContext.sortField;
+
   if (queryContext.statusFilter) {
     filters.push({ match: { 'monitor.status': queryContext.statusFilter } });
   }
 
-  const body = {
-    size: 0,
+  const sort = sortField
+    ? [
+        {
+          [sortField]: {
+            order: queryContext.sortDirection,
+          },
+        },
+      ]
+    : undefined;
+
+  if (sortField === 'summary.down' && sort) {
+    sort.push({
+      '@timestamp': {
```

Comment:
Using `@timestamp` as a 'default` sort doesn't make sense, because the order of the elements in the table will be unstable, which would be visually frustrating. It makes more sense to sort based on stable items like `monitor.id` as a default, or `monitor.name`